tests/phpunit/helpers/class-wc-helper-product.php

```diff
@@ -345,23 +345,6 @@ public static function create_attribute( $raw_name = 'size', $terms = [ 'small'
 		return $return;
 	}
 
-	/**
-	 * Delete an attribute.
-	 *
-	 * @param int $attribute_id ID to delete.
-	 *
-	 * @since 2.3
-	 */
-	public static function delete_attribute( $attribute_id ) {
```

Comment:
Removed this because isn't used anywhere and it's querying the database directly. It seems [`wc_delete_attribute`](https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce/includes/wc-attribute-functions.php#L661) should be used instead.